docs/whats_new/v0.7.0.md

```diff
@@ -0,0 +1,145 @@
+# Version 0.7.0
+
+We're happy to announce the AutoGluon 0.7 release. This release contains a new experimental module `autogluon.eda` for exploratory
+data analysis. Meanwhile, 0.7 offers enhancements to Tabular, MultiModal, and Time Series
+modules, along with many quality of life improvements and fixes.
+
+As always, only load previously trained models using the same version of AutoGluon that they were originally trained on.
+Loading models trained in different versions of AutoGluon is not supported.
+
+This release contains [**170** commits from **19** contributors](https://github.com/autogluon/autogluon/graphs/contributors?from=2023-01-10&to=2023-02-16&type=c)!
+
+See the full commit change-log here: https://github.com/autogluon/autogluon/compare/v0.6.2...v0.7.0
+
+Special thanks to @MountPOTATO who is the first time contributors to AutoGluon this release!
+
+Full Contributor List (ordered by # of commits):
+
+@Innixma, @zhiqiangdon, @yinweisu, @gradientsky, @shchur, @sxjscience, @FANGAreNotGnu, @yongxinw, @cheungdaven,
+@liangfu,
+@tonyhoo, @bryanyzhu, @suzhoum, @canerturkmen, @giswqs, @gidler, @yzhliu, @Linuxdex and @MountPOTATO
+
+This version supports Python versions from 3.8 to 3.10 and dropped the support of Python 3.7. 
```

Comment:
This should be in `Core` changes that we support Python 3.10, it should be very visible.

Reply:
Added in `Core Changes` section

Reply:
Added